test/bio_enc_test.c

```diff
@@ -41,7 +41,8 @@ static int do_bio_cipher(const EVP_CIPHER* cipher, const unsigned char* key,
     BIO *b;
     static unsigned char inp[BUF_SIZE] = { 0 };
     unsigned char out[BUF_SIZE], ref[BUF_SIZE];
-    int i, lref, len;
+    int i, lref;
+    size_t len;
```

Comment:
Hmmm, it was a complaint about a negative return and a signed variable.  I'll revert this one since the code seems correct as it was.